pennylane/devices/default_clifford.py

```diff
@@ -210,6 +251,64 @@ def circuit():
         Maintaining and working with this tableau representation instead of the complete state vector
         makes the calculations of increasingly large Clifford circuits more efficient on this device.
 
+    .. details::
+        :title: Probabilities for Basis States
+        :href: clifford-probabilities
+
+        As the ``default.clifford`` device supports executing quantum circuits with a large number of qubits,
+        we restrict the ability to compute the ``analytical`` probabilities for `all` computational basis
+        states at once to maintain computational efficiency when system size scales beyond ``20`` qubits.
+
+        As long as number of qubits are below this limit, one can simply use the :func:`qml.probs <pennylane.probs>`
+        with its usual arguments and probabilities for the specified target states would be computed and returned.
+        We test this for a circuit that prepares the ``n``-qubit Greenberger-Horne-Zeilinger state (GHZ state) state.
+        This means that the probabilities for the basis states :math:`|0\rangle^{\otimes n}` and
+        :math:`|1\rangle^{\otimes n}` should be :math:`0.5`, and for :math:`0.0` for the rest.
+
+        .. code-block:: python
+
+            import pennylane as qml
+            import numpy as np
+            dev = qml.device("default.clifford", target_states = basis_states)
```

Comment:
Mind defining `basis_states`?

Reply:
Sorry, I forgot to remove it from here we have now removed the `target_states` keyword.